listenbrainz/tests/integration/test_user_timeline_event_api.py

```diff
@@ -815,3 +815,123 @@ def test_get_cb_review_events(self, mock_requests):
         self.assertEqual(metadata_3["rating"], review_event_3["metadata"]["rating"])
         self.assertEqual(metadata_3["text"], review_event_3["metadata"]["text"])
         self.assertEqual(review_mbid_3, review_event_3["metadata"]["review_mbid"])
+
+    def test_personal_recommendation_writes_to_db(self):
+        # Let's create 2 users, who follow the request sender
+        user_one = db_user.get_or_create(2, "riksucks")
```

Comment:
In this test, it would be nice to test that the personal recommendation is still after the user has unfollowed the user who recommended. Whether we want to show those or not, may be a choice but the aim of the test is to ensure that case doesn't crash the page.